### DISCUSSION TITLE
Hospital License Information in Header - (OH registers but does not license hospitals)

### DISCUSSION THREAD

**Discussion by foustsWCH** [CMSgov/hospital-price-transparency#27] [Q&A] [answered]
Ohio does not currently license hospitals (this is changing soon but not fully in place yet.)  However, Ohio does require to submit an annual registration report and hospitals are issued a registration number.   

https://odh.ohio.gov/know-our-programs/hospital-registration/hospitalregistration

Should the hospital license field in the MRF header be blank (since OH does not have hospital licenses) or should it reflect the registration number?

**Comment by RidenourK** [comment, marked as answer]
@foustsWCH Thank you for your question. 

In this instance, you may encode the registration number under the “license_number | [state]” column header or you may leave it blank. Please refer to the “[CSV documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV)” and the “[CSV Examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples/CSV)” on the [Hospital Price Transparency – Data Dictionary GitHub repository](https://github.com/CMSgov/hospital-price-transparency) for more information about how to encode data for this data element.
